graphql/helpers/token.js

```diff
@@ -18,7 +18,7 @@ module.exports.createJWTToken = async function createJWTToken ({ email, jwtSecre
 
 module.exports.populateUserFromJWT = function populateUserFromJWT ({ jwtSecret }) {
   return async function populateUserFromJWTMiddleware(req, res, next) {
-    const jwtToken = req.headers.authorization?.replace(/^Bearer /, '')
+    const jwtToken = req.headers.authorization?.replace(/^Bearer\s*/, '')
```

Comment:
```suggestion
    const jwtToken = req.headers.authorization?.replace(/^Bearer\s+/, '')
```

Je pense qu'il faut au minimum 1 espace non?

Reply:
Oui très juste 🙂